split-webapp/split/src/components/TransactionList.js

```diff
@@ -84,7 +86,14 @@ class TransactionList extends React.Component {
         </ExpansionPanelSummary>
         <ExpansionPanelDetails className="Payments">
           <div className="PaymentsTitle">
-            <PaymentIcon className="PaymentHeaders" />
+            <div className="PaymentHeaders">
+              <Link
+                to={{
+                  pathname: "split",
+                  bill
+                }}><EditIcon /></Link>
```

Comment:
I think I did do it this way because I saw an example that passed a custom property. If we pass a `state` property then the implementation would need to check for the existence of a `this.location.state` before defining `editBill` to avoid a null exception.

Reply:
Ah cool. What you've got looks good then.